src/crawlee/models.py

```diff
@@ -19,7 +19,7 @@ class BaseRequestData(BaseModel):
 
     model_config = ConfigDict(populate_by_name=True)
 
-    url: Annotated[str, Field(min_length=1)]
+    url: Annotated[HttpUrl, Field()]
```

Comment:
I'd wager the `Field()` annotation is redundant here. But I'm not sure myself if I'd prefer to keep it or not.

Reply:
pylance is against removing it, so let's leave it there